base/data/nlu/address.md

```diff
@@ -0,0 +1,266 @@
+## intent:address
+- LYNDHURST
+- 1345b machaba drive, mofolo village
+- Potchfstroom
+- Soweto,protea south
+- 5337 selosesha ext3 thaba nchu
+- joburg cbd
+- Bergview
+- Scottsville
+- Blouberg
+- Soweto ,tladi
+- Tladi soweto
+- Simon's town
+- Molapo ext,  soweto
+- Sir lowrys pass
+- Imbali bb3
+- Allemansdrift
+- Masilo
+- River club
+- Soulsville
+- Lyhndurst
+- Bothsig
+- I'm staying at vosloorus Sotho section
+- Dodsonvill
+- Chlookop ext2 farraday st
+- Brixton
+- Sibeko,  no 11 mshumi street. Mount Ayliff 4835
+- Soshanguvhe
+- Scottville
+- Ladybranf
+- Tshiawelo
+- Rondebosch east
+- Tshiwelo
+- Buffalo city
+- Sir lowry pass
+- Sea cow lake
+- Norwood , elsies river
+- Cedarville
+- No 69 Weko str
+- Clayville
+- numbi trust
+- Cuyler street town
+- Soweto
+- 📍Soweto
+- Sibasa
+- Zebediela
+- Soweto, zola
+- TEMBIDS
+- Mfekayi
+- 239 phadima section; katlehong next to kathorus college
+- +MALBORO
+- Akobe
+- Ivory
+- Johannesbhsg
+- Lebowakgomo
+- Mamelodi east sns.
+- Magogoe koikoi
+- SHAKADZA
+- BRITSTOWN
+- Port st Johns
+- Vall
+- Haddon
+- Tigane
+- Lochnerhof
+- Moloto block 20
+- Soweto naledi
+- Toti
+- Zola2
+- Klerksdorp,
+jouberton -
```

Comment:
line break

Reply:
This is a multi-line message from a user. I didn't see any errors/warnings during training, so I assumed that this was the correct way of representing it in the data.